src/doc/playback.cpp

```diff
@@ -119,10 +119,7 @@ frame_t Playback::nextFrame(frame_t frameDelta)
       handleMoveFrame(step);
     handleEnterFrame(step, false);
 
-    if (frameDelta > 0)
-      --frameDelta;
-    else if (frameDelta < 0)
-      ++frameDelta;
+    frameDelta -= step;
```

Comment:
I think in our tests and code `frameDelta` is always +1 or -1, but what happen if it's +2 or -2? (it looks like the while could be infinite). Is it necessary to replace the original code?

Reply:
Oh sorry, it looks fine, in some way so something like frame += frameDelta, I think this is fine.